experimental/PropertyDDS/packages/property-binder/test/data_binder/property_element.spec.js

```diff
@@ -648,4 +649,13 @@ describe('Property element', function() {
     expect(propElem.getChild(['myReference2', 'aNumber'], RESOLVE_NO_LEAFS).getValue()).toEqual(42);
   });
 
+  it('getChild with special characters in tokenized path', function() {
+    const myData = PropertyFactory.create(PrimitiveChildrenTemplate.typeid);
+    workspace.root.insert('"/"', myData);
+
+    const propElem = new PropertyElement(workspace.root);
+    expect(propElem.getChild(['"/"', 'aString']).isValid()).toEqual(true);
+    expect(propElem.getChild(['"/"', 'aString', '/']).getProperty()).toEqual(workspace.root);
```

Comment:
This is not the expected behaviour. In a tokenized path, a '/' should resolve to a property with that name or to undefined if it does not exist. It should not return to the root. In addition to that, a '/' as root selector is only valid as first entry in a path.